File: Java Programs/College/IT_X/Stack.java
import java.util.Scanner;
class Stack{
	int []a,top,max;
	Stack(){
	  a=new int [50];
	  top=-1;
	  max=n;
	}
	void push(int item){
	  if(top==max-1)
	    System.out.println("Stack is full.");
	  else
	    a[++top]=item;
	}
	int pop(){
	  if(top==-1)
	    System.out.println("Stack is empty.");
	  else{
	    System.out.println("The popped element: "+a[top--]);
	  }
	}
}
class MyStack{
	int top=-1;
	public static void main(String []args){
	  int n;
	  Scanner inp= new Scanner(System.in);
	  while(true){
	    System.out.println("Press\n [1]To Push\n [2]To Pop\nEnter choice: ");
	    int ch=inp.nextInt();
	    switch(ch){
	      case 1:
		System.out.println("Enter the value to push into the stack: ");
		n=inp.nextInt();
	        push(n);
	    }
	} 
	}
}

File: C Programs/IT_291_391_C/stack.c
#include<stdio.h>
#define max 10
int top=-1;int a[max];
void push(int item)
{
if(top==max-1)

printf("the stack is full");
else
{
top=top+1;
a[top]=item;
}
}
void pop()
{
if(top==-1)
{
printf("the stack is empty");
}
else
{
printf("%d",a[top]);
top=top-1;
}
}
void main()
{
int i,n,r;
printf("enter the decimal no");
scanf("%d",&n);
while( n>0)
{
r=n%2;
push(r);
n=n/2;
}
printf("the binary no is");
while(top>=0)
pop();
}


